### PR TITLE
feat: Disable Connect to Skill menu action inside skill

### PR DESCRIPTION
**Pull request by srinaath**
### Description
The Connect to skill action should be enabled only in a Root Bot. A skill cannot connect to another skill when its not in the Root Bot context

Screen Shot 2020-10-29 at 10 00 03 AM

Refs #3954 
###

### CODE REVIEW

File: Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/EdgeMenu/createSchemaMenu.tsx [hunk 1]
```diff
@@ -174,13 +176,48 @@ interface ActionMenuOptions {
...
+    let additionalProps: Partial<IContextualMenuItem> = {};
+    const disabledAction = forceDisabledActions.find((action) => action.kind === subMenuItem.key);
+    if (disabledAction) {
+      additionalProps = {
+        title: disabledAction.reason,
+        disabled: true,
+        onRender: (item) => {
+          const calloutProps = { gapSpace: 0 };
+          const tooltipId = `tooltip-${disabledAction.kind}`;
+          const hostStyles: Partial<ITooltipHostStyles> = {
```
Comment (hatpick) [inline Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/EdgeMenu/createSchemaMenu.tsx:192]: You can extract this css outside the callback

File: Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/EdgeMenu/createSchemaMenu.tsx [hunk 2]
```diff
@@ -174,13 +176,48 @@ interface ActionMenuOptions {
...
+            root: {
+              display: 'flex',
+              justifyContent: 'flex-start',
+              alignItems: 'center',
+              padding: '0px 7px',
+              height: '38px',
+              background: '#F3F2F1',
```
Comment (hatpick) [inline Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/EdgeMenu/createSchemaMenu.tsx:199]: use NeutralGray equivalent of this color, helps in the future with theming (Gray20)
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/neutrals

File: Composer/packages/types/src/shell.ts [hunk 3]
```diff
@@ -41,6 +41,11 @@ export type BotSchemas = {
   diagnostics?: any[];
 };
 
+export type DisabledMenuActions = {
+  kind: string;
```
Comment (hatpick) [inline Composer/packages/types/src/shell.ts:45]: is the type SDKKinds?

Comment (srinaath) [reply Composer/packages/types/src/shell.ts:45]: It is of that type

**Review by hatpick — APPROVED** [review]

**Review by hatpick — APPROVED** [review]

**Pull request merged**